atomic_reactor/__init__.py

```diff
@@ -32,6 +35,28 @@ def format(self, record):
         return super(ArchFormatter, self).format(record)
 
 
+class EncodedStream(object):
+    def __init__(self, fileno, encoding):
+        self.binarystream = fdopen(dup(fileno), 'wb')
```

Comment:
Tim and I couldn't figure out how to force stderr to encode UTF-8 for output.  Tim stumbled upon this trick.  Suggestions welcome? :-)
Not sure what would be the symptom of an overlapping write issues.  My log output seems fine.